lib/ecl/ecl_smspec.cpp

```diff
@@ -1250,6 +1226,36 @@ static bool ecl_smspec_fread_header(ecl_smspec_type * ecl_smspec, const char * h
 }
 
 
+/******************************************************************/
+
+static const smspec_node_type * ecl_smspec_get_var_node( const node_map& mp, const char * var) {
```

Comment:
Instead of using `static` you should use an anonymus namespace - that is the C++ way:
```
namespace {
    
    const smspec_node_type * ecl_smspec_get_var_node( ) {  }

}  
```

Reply:
Or - maybe not; the `static` keyword seems to have back in favor also for C++: https://stackoverflow.com/questions/154469/unnamed-anonymous-namespaces-vs-static-functions